metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -1095,9 +1118,15 @@ void handleBrokerFenced(int brokerId, List<ApiMessageAndVersion> records) {
         }
         generateLeaderAndIsrUpdates("handleBrokerFenced", brokerId, NO_LEADER, records,
             brokersToIsrs.partitionsWithBrokerInIsr(brokerId));
-        records.add(new ApiMessageAndVersion(new FenceBrokerRecord().
-            setId(brokerId).setEpoch(brokerRegistration.epoch()),
-            FENCE_BROKER_RECORD.highestSupportedVersion()));
+        if (featureControl.metadataVersion().brokerRegistrationChangeRecordSupported()) {
+            records.add(new ApiMessageAndVersion(new BrokerRegistrationChangeRecord().
+                    setBrokerId(brokerId).setBrokerEpoch(brokerRegistration.epoch()).setFenced((byte) 1),
```

Comment:
Can we define the `(byte) 1` constant somewhere? This is used a in few places.

Reply:
ok